.github/workflows/test-and-publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       # - name: Client - audit
       #   run: cd client && npm run test:audit
       - name: Client - format test
-        run: cd client && npm run test:format
+        run: npm run test:format
```

Comment:
Remove the `Client` prefix from the test name. Maybe make it the first test, so doesn't look weird w others